Lib/test/test_tokenize.py

```diff
@@ -1937,25 +1937,34 @@ def test_string(self):
     """)
 
         self.check_tokenize('f"abc"', """\
-    STRING     'f"abc"'      (1, 0) (1, 6)
+    FSTRING_START 'f"'          (1, 0) (1, 2)
+    FSTRING_END 'abc'         (1, 2) (1, 5)
     """)
 
         self.check_tokenize('fR"a{b}c"', """\
-    STRING     'fR"a{b}c"'   (1, 0) (1, 9)
+    FSTRING_START 'fR"'         (1, 0) (1, 3)
+    FSTRING_MIDDLE 'a'           (1, 3) (1, 4)
```

Comment:
FYI: the reason you cannot align this is that this is comparing it against the stringify versions of the tokens via this code:

https://github.com/python/cpython/blob/ef000eb3e2a8d0ecd51b6d44b390fefd820a61a6/Lib/test/test_tokenize.py#L34

which is a suboptimal way of doing the job for sure :sweat_smile:

Reply:
noted!